server/models/06-bhima.sql

```diff
@@ -332,7 +332,7 @@ INSERT IGNORE INTO `transaction_type` (`id`, `text`, `type`, `fixed`) VALUES
   (2, 'VOUCHERS.SIMPLE.CASH_PAYMENT', 'income', 1),
   (3, 'VOUCHERS.SIMPLE.CONVENTION_PAYMENT', 'income', 1),
   (4, 'VOUCHERS.SIMPLE.SUPPORT_INCOME', 'income', 1),
-  (5, 'VOUCHERS.SIMPLE.TRANSFER', 'other',  1),
+  (5, 'VOUCHERS.SIMPLE.TRANSFER', 'income',  1),
```

Comment:
This is a breaking change, and we should talk about it.

Why are all transfers considered income?  I think the majority of account to account transfers at Vanga (for example, from the auxiliary cashbox to the primary cashbox) are marked as "transfers".  Same for transfers to and from the bank.

Why is a transfer an income?

Reply:
Regarding the change of transfer type transaction, I had chosen to modify the type to allow the analysis of transfers coming from the auxiliary cash registers of the receipts, but since this will have repercussions on the other production sites, **_I have the following proposal: add in the interface of the cashflow report the possibility of considering the transfers as being receipts and the default value will be false._**

Reply:
But regarding the cashflow report, I just think to add the option to consider the transfer in the income category, like that we will not need to add new types of transaction_type

Reply:
Yes, I'm okay with this for IMCK.